src/WebJobs.Script.WebHost/Models/HostStatus.cs

```diff
@@ -30,5 +31,29 @@ public HostStatus()
         /// </summary>
         [JsonProperty(PropertyName = "errors", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public Collection<string> Errors { get; set; }
+
+        /// <summary>
+        /// Gets or sets the web host settings.
+        /// </summary>
+        [JsonProperty(PropertyName = "webHostSettings", DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public WebHostSettings WebHostSettings { get; set; }
+
+        /// <summary>
+        /// Gets or sets the processId for the host.
+        /// </summary>
+        [JsonProperty(PropertyName = "processId", DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public int ProcessId { get; set; }
+
+        /// <summary>
+        /// Gets or sets whether the process has a debugger attached or not.
+        /// </summary>
+        [JsonProperty(PropertyName = "isDebuggerAttached", DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public bool IsDebuggerAttached { get; set; }
+
+        internal static string GetAssemblyFileVersion(Assembly assembly)
```

Comment:
I think this was a merge mistake. You moved it from here back in August https://github.com/Azure/azure-webjobs-sdk-script/commit/0c50aa890f7a5a81aeb7c24676ec47647e7efc92.
Removed.